Corgibytes.Freshli.Cli/Factories/ICommandRunnerFactory.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Corgibytes.Freshli.Cli.CommandOptions;
+using Corgibytes.Freshli.Cli.CommandRunners;
+
+namespace Corgibytes.Freshli.Cli.Factories
+{
+    public interface ICommandRunnerFactory
+    {
+        ICommandRunner<ScanCommandOptions> CreateScanCommandRunner(ScanCommandOptions scanOptions);
+        ICommandRunner<AuthCommandOptions> CreateAuthCommandRunner();
```

Comment:
- Yes, we can remove the auth command until implemented
- We need interfaces because that is the way we can decouple implementation from contracts. I do not want to inject or create concrete objects.

Reply:
I am thinking that would be good to keep the AuthCommand so it is easy to see how to add new commands. It does not hurt, since I throw a "NotImplementedException" if people tries to run it.